python/rpdk/typescript/codegen.py

```diff
@@ -251,13 +257,22 @@ def _build(self, base_path):
 
         LOG.warning("Starting build.")
         try:
+            if os.path.exists("/bin/bash"):
+                shell = "/bin/bash"
+                shell_arg = "-c"
+            # On windows get command line interpreter stored in COMSPEC environment variable e.g. c:\Windows\System32\cmd.exe
+            elif os.environ.get("comspec"):
```

Comment:
If not `/bin/bash/` its windows?  Isn't there a better way to determine if we are on windows. `sys.platform`? [docs](https://docs.python.org/3/library/sys.html#sys.platform)

Should we throw an error if we are on non-Windows without /bin/bash?

Reply:
Switched to using the python subprocess default shell for other non Windows OSes.  On windows the python subprocess shell is inconsistent and I have to build the command by hand.